tests/test_1.py

```diff
@@ -2,6 +2,7 @@
 import requests as r
 
 class TestClass(unittest.TestCase):
+    

```

Comment:
Workflows looks nice

The only notice I have is about tests:
In unittests we never connect to some existing urls (localhost of others). If you want to do the **unit** testing of your endpoints - use `from fastapi.testclient import TestClient`
If you wanted to test with this function that the code was successfully deployed and run - unittests is not the right place. The more convenient way to do this is:
* create a /health endpoint 
* call it from ci/cd scripts after your application was deployed and run. And then check that it returns success.

But it is not required to do it exactly this way.
Btw, don't forget about the deployment of the application. You can start to investigate AWS API Gateway.